pkg/agent/plugin/workloadattestor/k8s/sigstore/sigstore.go

```diff
@@ -91,7 +91,13 @@ func New(cache Cache, logger hclog.Logger) Sigstore {
 	}
 }
 
-func defaultCheckOptsFunction(rekorURL url.URL) (*cosign.CheckOpts, error) {
+func defaultCheckOptsFunction(rekorURL url.URL, privateDeployment ...bool) (*cosign.CheckOpts, error) {
```

Comment:
privateDeployment = True means EnforceSCT = True, which means this is not a private deployment
rename to enforceSCT or useSCT, etc.